packages/@okta/vuepress-site/.vuepress/nav/newNav/guides.js

```diff
@@ -0,0 +1,232 @@
+// the navigation that is shown across the guides pages
+
+module.exports = [
+  {
+    title: "Guides",
+    subLinks: [
+      { title: 'Guides overview', path: '/docs/guides/'},
+      {
+        title: "Basics",
+        subLinks: [
+          {
+            title: "Quickstart: Signing in your first user",
+            guideName: "quickstart"
```

Comment:
Not familiar with this so asking, any reason this file uses guideNames but all 3 other uses paths?

Reply:
yes it's something that was done previously by other developers, guides include some additional logic so I didn't want to touch it